docs/view_components/menu_bar.rst

```diff
@@ -6,28 +8,27 @@ Menu bar is a set of options linked to actions, or submenus.
 Attributes
 ^^^^^^^^^^
 
-* The following are the menus available in the menu bar:
+**The following are the menus available in the menu bar:**
     #. File
     #. Scenarios
     #. Preferences
     #. Help
-* *File* has the following submenus or options:
+***File* has the following submenus or options:**
```

Comment:
We could add this directly in the first list also? Then we loose one indent level, so maybe not a good idea. To explore